elements/pfe-modal/src/pfe-modal.js

```diff
@@ -159,6 +159,7 @@ class PfeModal extends PFElement {
 
     this.isOpen = true;
     // Reveal the container and overlay
+    this.removeAttribute("hidden");
```

Comment:
@LyndseyR - Do we want to add a hidden attribute back to the modal light DOM when the modal is closed?

Reply:
I was pondering this - the challenge is, some modals (maybe most) will have the link that triggers the modal inside a slot so we don't _want_ hidden on the host.  We'd need some flag that says it had hidden on instantiation or something.

Reply:
How important do you think adding it back is?  The content is hidden in the Shadow DOM, so it's only relevant for before the component upgrades.